src/pages/order/order.ts

```diff
@@ -70,11 +70,14 @@ export class OrderPage {
   }
 
   public ConfirmOrder() {
-    // Service GetDate
+    // Service GetDatei
     let dateObj = new Date();
     let month = (dateObj.getUTCMonth() + 1).toString(); //months from 1-12
+    console.log(month.length.toString());
+
+
     // If month/day doesn't have two digits
-    if (month != "10" || "11" || "12" ){
+    if (month.length != 2){
```

Comment:
lol

Reply:
lol yeah. for some reason you cant do month != "11". its an error